### PR TITLE
Skateboards can take you to the afterlife

### PR DESCRIPTION
**Pull request by Tallvisit**
## About The Pull Request

Makes it so when you use the Suicide command with any of the skateboards in hand, you attempt the forbidden double kickflip and die.

## Why It's Good For The Game

Adds a way for people to kill themselves while making it look like they are still Rad.

Also makes it so when committing suicide with a skateboard there is a 3% chance for an additional message to appear, and a mood boost to any onlookers.

## Changelog
:cl:
add: Added suicide text to all Skateboards, and gave them a 3% chance to do something extra.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -534,6 +534,10 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	///The vehicle counterpart for the board
 	var/board_item_type = /obj/vehicle/ridden/scooter/skateboard
 
+/obj/item/melee/skateboard/suicide_act(mob/living/carbon/user)
+	user.visible_message("<span class='suicide'>[user] begins attempting to preform a double kickflip! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	return BRUTELOSS
```
Comment (Superyodeler) [inline code/game/objects/items/weaponry.dm:539]:
```suggestion
/obj/item/melee/skateboard/suicide_act(mob/living/carbon/user)			// Wasp Edit Begin - Skateboard to the afterlife
	user.visible_message("<span class='suicide'>[user] begins attempting to preform a double kickflip! It looks like [user.p_theyre()] trying to commit suicide!</span>")
	return BRUTELOSS			// Wasp Edit End
```

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Looks good, just needs some margularization comments

**Review by Superyodeler — APPROVED** [review]
Looks good!

**Comment by Tallvisit** [comment]
Sick, time for victory Mac and Cheese...




Marg and Cheese...

**Comment by Tallvisit** [comment]
Right, this should be ready to hit the shelves, unless I missed something, I made that harder then it should have been...

Noted changes is just developer notes, an added moodlet, and a chance for the skateboard suicide to trigger this moodlet for onlookers.

**Review by MarkSuckerberg — APPROVED** [review]

**Comment by Tallvisit** [comment]
Merged! Now everyone can suicide round start for a chance at that gamer mood boost for everyone else.

**Comment by ADepressedPyroshark** [comment]
> 
> 
> Merged! Now everyone can suicide round start for a chance at that gamer mood boost for everyone else.

brb setting up suicide macro to automate the process

**Pull request merged**